ocp4/CMakeLists.txt

```diff
@@ -5,5 +5,6 @@ endif()
 
 set(PRODUCT "ocp4")
 set(DISA_SRG_TYPE "os")
+set(PRODUCT_REMEDIATION_LANGUAGES "ignition;kubernetes")
```

Comment:
Yeah... `product.yml` seems like a good place for this kind of into.

But, I may not be that easy.
The python scripts consume the data in `product.yml`, but AFAIK the CMake scripts don't actually load its contents. And the remediation languages are important in CMake to correctly order the build scripts and connect their dependencies.

Reply:
It doesn't seem so easy to load data from a `.yml` as a CMake variable, I'm open to suggestion on how to do it.

Reply:
Darn! Was hoping it would be easy. I was hoping that we actually could do this without adding a CMake variable at all, but I guess we are using the XSLT to generate the files at point.

Reply:
> but I guess we are using the XSLT to generate the files at point.

@redhatrises I don't know what files you are talking about.